test/exporter/test_quantify_scheduler_exporter.py

```diff
@@ -63,12 +64,16 @@ def test_export(self):
                 [
                     unittest.mock.call(theta=FloatEq(math.degrees(math.pi)), phi=FloatEq(0), qubit="q[0]"),
                     unittest.mock.call(
-                        theta=FloatEq(math.degrees(1.23)), phi=FloatEq(math.degrees(math.pi / 2)), qubit="q[2]"
+                        theta=FloatEq(round(math.degrees(1.23), DEG_PRECISION)),
+                        phi=FloatEq(math.degrees(math.pi / 2)),
```

Comment:
Two questions here:
1) Why don't we round `phi`?
2) We wouldn't need to round `theta`, right? Because we know already that `1.23` uses 2 decimals and we allow 5. But, anyway, I would leave it as it makes the code more consistent.

Reply:
I'm rounding after converting to degrees, and for the test the pi/2 is converted nicely to 90 degrees, so it does not need to be rounded.